esm-samples/jsapi-deno/README.md

```diff
@@ -0,0 +1,37 @@
+# ArcGIS API for JavaScript in Deno
+
+Integrating [Deno](https://deno.land/) with [`@arcgis/core`](https://www.npmjs.com/package/@arcgis/core) can be done by using [TypeScript](https://www.typescriptlang.org/) with no compile steps.
+
+## IdentityManager
+
+You will also want to disable the [`IdentityManager`](https://developers.arcgis.com/javascript/latest/api-reference/esri-identity-IdentityManager.html) using [`config.request`](https://developers.arcgis.com/javascript/latest/api-reference/esri-config.html#request) so it doesn't attempt to load DOM-related JavaScript.
+
+```js
+import esriConfig from "npm:@arcgis/core/config.js";
+esriConfig.request.useIdentity = false;
+```
+
+## Deno Server
+
+These samples use the [oak](https://deno.land/x/oak@v11.1.0) middleware for building a server with Deno. Please refer to the oak documentation for more details.
```

Comment:
Oak claims to only support node 16.5 or later. Maybe add that as a short note?

Reply:
I think users should refer to the oak doc for that. If it changes to 18 or something in a few months, we need to update or doc

Reply:
Also seems less relevant in a Deno sample.

Reply:
Sounds good, referring to the doc.